Corona-Warn-App/src/main/java/de/rki/coronawarnapp/storage/LocalData.kt

```diff
@@ -524,17 +492,6 @@ object LocalData {
         )
     }
 
-    fun teletan(value: String?) = getSharedPreferenceInstance().edit(true) {
```

Comment:
Can you explain all of these can be removed now? Was it just dead code?

Reply:
teletan and testGUID were both used when test scanning / tan entering and test registration were different steps and ui screens, since they have been merged into their respective single screen we do not need to persist this data anymore but we still did for quite a while without ever using it